recipe-server/client/control_new/components/pages/MissingPage.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function MissingPage() {
+  return (
+    <div className="page-404">
+      <h1>404</h1>
+      <h3>Page not found.</h3>
```

Comment:
Nit: [MDN recommends avoiding skipping heading levels](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). Since this isn't really a heading anyway, this should probably be a `p` tag anyway.